mne/io/edf/edf.py

```diff
@@ -1223,3 +1219,20 @@ def _get_edf_default_event_id(descriptions):
     mapping = dict((a, n) for n, a in
                    enumerate(sorted(set(descriptions)), start=1))
     return mapping
+
+
+def _get_annotations_gdf(edf_info, sfreq):
+    onset, duration, desc = list(), list(), list()
+    events = edf_info.get('events', None)
+    # Annotations in GDF: events are stored as the following
+    # list: `events = [n_events, pos, typ, chn, dur]` where pos is the
+    # latency, dur is the duration in samples. They both are
+    # numpy.ndarray
+    if events is not None and events[1].shape[0] > 0:
+        onset = events[1] / sfreq
+        duration = events[4] / sfreq
+        desc = [GDF_EVENTS_LUT[key]
+                if key in GDF_EVENTS_LUT.keys() else 'Unknown'
```

Comment:
GDF_EVENTS_LUT.keys() -> GDF_EVENTS_LUT

if you put the .keys() the "in" cost you a O(n) vs O(1) (list vs hash table)